arch/arm/soc/st_stm32/stm32f3/Kconfig.series

```diff
@@ -13,6 +13,7 @@ config SOC_SERIES_STM32F3X
 	select SYS_POWER_LOW_POWER_STATE_SUPPORTED
 	select CPU_HAS_FPU
 	select CPU_HAS_SYSTICK
+	select CPU_HAS_MPU
```

Comment:
Since MPU is not suported on all SoCs of stm32f series, this should be moved (if indeed supported) under Kconfig.defconfig.stm32f3....